Worktrolly/worktrolly
# Feat/rewards page

### Functionality:
Implementation of rewards page

### Solution:
1. Created API's to manage business logic.
2. Created rewards interface to use it as constructor.
3. Created Ui including forms to take inputs and cards to show reward info.

## WtId:

### Risk level:

### How to test:
1.Open Emulator 
2. Go to my organization
3. Click on  manage rewards.
4. Click on add reward and create a reward.
5. You will see a reward card added in the page.
